store/tx_manager_test.go

```diff
@@ -410,19 +410,13 @@ func TestTxManager_BumpGasUntilSafe_erroring(t *testing.T) {
 			ethMock.Register("eth_getTransactionReceipt", nonConfedReceipt)
 			ethMock.RegisterError("eth_getTransactionReceipt", "FUBAR")
 		}, false, true},
-		{"early conf, no error", (safeAt + 1), func(ethMock *cltest.EthMock) {
+		{"early conf", (safeAt + 1), func(ethMock *cltest.EthMock) {
 			ethMock.Register("eth_getTransactionReceipt", confedReceipt)
-			ethMock.Register("eth_getTransactionReceipt", nonConfedReceipt)
-		}, true, false},
-		{"early conf, later error", (safeAt + 1), func(ethMock *cltest.EthMock) {
-			ethMock.Register("eth_getTransactionReceipt", confedReceipt)
-			ethMock.RegisterError("eth_getTransactionReceipt", "FUBAR")
```

Comment:
Having removed the `getTransactionReceipt` error mock for the "later error" example, it was identical to the earlier example so I just condensed them into one. I also confirmed that these changes pass on master, without the additions to ethMock, it looks like they stopped being used but the tests did not check that all expectations were called.